sidebars.js

```diff
@@ -28,13 +28,18 @@ module.exports = {
       items: [
         {
           type: 'category',
-          label: '⚙️ Setting up Great Expectations',
+          label: 'Setting up Great Expectations',
```

Comment:
I'm not super well versed in the planned design changes we're making to docs after the initial release so I may not be the best reviewer for this particular file.

Looks good to me when I render it in the preview though!

Reply:
I loved these little icons, but people with more *ahem* taste than me disagreed. So we're removing them.